index.js

```diff
@@ -59,13 +60,22 @@ module.exports = function(source) {
     }
   });
 
+  manualImports = manualImports.map(function (it) { return utils.lookup(it, options.paths, '', true); });
```

Comment:
PathCache does [stylus lookups](https://github.com/shama/stylus-loader/blob/master/lib/pathcache.js#L88-L108) as well as webpack resolution if the stylus lookups fail. This line shouldn't be needed.